src/modules/auth/views/activation-view.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <div class="auth-view-container">
+    <AuthLoader
+      :is-visible="sendActivationTokenMutation.isPending.value"
+    />
+    <div
+      v-if="sendActivationTokenMutation.isError.value"
+      class="auth-view-container--error"
+    >
+      <v-icon>mdi-alert-circle</v-icon>
+      {{ errorMessage }}
+    </div>
+    <div
+      v-if="sendActivationTokenMutation.isSuccess.value"
+      class="auth-view-container--success"
+    >
+      <header class="d-flex justify-center align-center">
+        <v-icon :size="28">mdi-check-circle</v-icon>
```

Comment:
Chyba nie zauważyłem wcześniej w projekcie implementacji v-icon, czy to jest jakaś zewnętrzna biblioteka? Jak tam ogarniane jest a11y? Czy ta ikona ma jakąś labelkę dla czytnków ekranowych aby user wiedział co t ojest, czy może jest w jakiś sposób aria-hidden ustawiony aby czytnki ignorowały ją? Są różne techniki a11y dla ikon

Reply:
To vuetify. Ikona dostaje aria-hidden na true.